pkg/email/system_intake_confirm_lcid.go

```diff
@@ -0,0 +1,139 @@
+package email
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"html/template"
+	"path"
+	"time"
+
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/easi-app/pkg/apperrors"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+type systemIntakeConfirmLCIDEmailParameters struct {
+	RequestName              string
+	LifecycleID              string
+	LifecycleExpiresAt       string
+	LifecycleScope           template.HTML
+	LifecycleCostBaseline    string
+	DecisionNextSteps        template.HTML
+	TRBRecommendation        string
+	RequesterName            string
+	SystemIntakeRequestLink  string
+	SystemIntakeAdminLink    string
+	ITGovernanceInboxAddress string
+	AdditionalInfo           template.HTML
+}
+
+func (sie systemIntakeEmails) SystemIntakeConfirmLCIDBody(
+	systemIntakeID uuid.UUID,
+	requestName string,
+	lifecycleID string,
+	lifecycleExpiresAt *time.Time,
+	lifecycleScope models.HTML,
+	lifecycleCostBaseline string,
+	decisionNextSteps models.HTML,
+	trbFollowUp models.SystemIntakeTRBFollowUp,
+	requesterName string,
+	additionalInfo *models.HTML,
+) (string, error) {
+	requesterPath := path.Join("governance-task-list", systemIntakeID.String())
+	adminPath := path.Join("governance-review-team", systemIntakeID.String(), "intake-request")
+
+	var expiresAt string
+	if lifecycleExpiresAt != nil {
+		expiresAt = lifecycleExpiresAt.Format("01/02/2006")
+	}
+
+	var trbFollowUpReadable string
+	switch trbFollowUp {
```

Comment:
I thought about making it a utility method but it's also only relevant to emails currently. The advantage to repeating it is that if the email contents drift for whatever reason then we can just change the verbiage in each place instead of worrying about the shared method.

If it's OK, I was thinking just leave it and we can break it out into its own method later if that makes sense.

Reply:
Sounds good to me.